src/_components/userPoints/UserPoints.tsx

```diff
@@ -1,30 +1,71 @@
-import React from 'react'
+import React from 'react';
 
-import { useToggle } from '../../_hooks';
+import { useStackNavigation, useToggle } from '../../_hooks';
+import { getAvatarByNameOrDefault } from '../../_utils';
+import { useGetFamilyMembers } from '../../onboarding/family/_queries';
 import { useGetMe } from '../../profile/_queries/useGetMe';
 import CardModal from '../../profile/CardModal/CardModal';
 import Typography from '../typography/Typography';
 import * as Styled from './style';
 
 const UserPoints = () => {
-  const { data: user, isLoading, isError } = useGetMe();
+  const navigation = useStackNavigation();
   const [cardModalVisible, toggleCardModalVisible] = useToggle(false);
 
-  if (isLoading || isError) return null;
-  const initials = `${user.firstName?.[0] || ''}${user.name?.[0] || ''}`;
-  return (
-    <>
-      <Styled.UserPointsButton activeOpacity={0.8} onPress={toggleCardModalVisible}>
-        <>
-          <Styled.PointsLabel allowFontScaling={false} color="neutral.0" fontStyle='bold' size="normal" >{user.points}</Styled.PointsLabel>
-          {!!initials && <Styled.AvatarContainer>
-            <Typography allowFontScaling={false} color="secondary.500" fontStyle='bold' size="xsmall">{initials}</Typography>
-          </Styled.AvatarContainer>}
-        </>
-      </Styled.UserPointsButton>
-      <CardModal isVisible={cardModalVisible} toggleIsVisible={toggleCardModalVisible} />
-    </>
-  )
-}
-
-export default UserPoints;
\ No newline at end of file
+  const { data: user, isLoading: isLoadingProfile, isError: isErrorProfile } = useGetMe();
+  const { data: familyMembers, isLoading: isLoadingFamilyMembers, isError: isErrorFamilyMembers } = useGetFamilyMembers();
+
+  if (isLoadingProfile || isLoadingFamilyMembers || isErrorProfile || isErrorFamilyMembers) {
+    return null;
+  }
+
+  if (familyMembers.length === 1) {
+    const initials = `${user.firstName?.[0] || ''}${user.name?.[0] || ''}`;
+
+    return (
+      <>
+        <Styled.UserPointsButton activeOpacity={0.8} onPress={toggleCardModalVisible}>
```

Comment:
will we always do it this way? The difference between single / family.
Maybe split it in different components? Use a FamilyHook that renders one or the other component and gets the shared data?